tests/pipeline/nodes/model/posenetv1/test_posenet.py

```diff
@@ -72,7 +73,9 @@ def test_different_models(self, posenet_model, person_image):
         assert posenet_model is not None
         output = posenet_model.run({'img': pose_image})
         expected_output = dict.fromkeys(['bboxes', 'keypoints', 'keypoint_scores',
-                                         'keypoint_conns'])
+                                         'keypoint_conns', 'bbox_labels'])
         assert output.keys() == expected_output.keys(), "missing keys"
         for i in expected_output.keys():
             assert len(output[i]) >= 1, "unexpected number of outputs for {}".format(i)
+        for label in output['bbox_labels']:
+            assert label == 'Person'
```

Comment:
Note: I made changes to the posenet test to include the bbox_labels so it won't fail. Also added a simple test for the created labels